kotlinx-coroutines-test/common/test/RunTestTest.kt

```diff
@@ -112,7 +110,7 @@ class RunTestTest {
                 it()
                 fail("unreached")
             } catch (e: UncompletedCoroutinesError) {
-                assertTrue((e.message ?: "").contains(name1))
+                assertContains(e.message ?: "", name1)
```

Comment:
Filed https://youtrack.jetbrains.com/issue/KT-53336/Improve-assertContains-and-friends-provide-negation-and-nullability-support